modules/osd-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+[id="osd-aws-privatelink-create-cluster.adoc_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create an AWS PrivateLink cluster using the `rosa` CLI.
+
+.Prerequisites
+
+You have completed the installation prerequisites.
+
+.Procedure
+
+
+Creating a cluster can take up to 40 minutes.
+
+. With AWS PrivateLink, you can create a cluster with a single availability zone (Single-AZ) or multiple availability zones (Multi-AZ).
++
+[IMPORTANT]
+====
+If you use a firewall, you must configure it so that {product-title} can access the sites that it requires to function.
+
+Refer to the AWS PrivateLink firewall prerequisites section for details.
+====
+** To create a Single-AZ cluster:
++
+[source,terminal]
+----
+$ rosa create cluster --private-link --cluster-name=<cluster-name> --machine-cidr=<VPC CIDR>/16 --subnet-ids=<private-subnet-id>
```

Comment:
Should we show --machine-cidr option here? We have no cidr and subnet validation yet, it will cause error cluster easily.

Reply:
And I think we'd better have a warning here that the machine-cidr should match the subnet cidr

Reply:
@dustman9000 @wgordon17  
Remove --machine-cidr=<VPC CIDR>/16 from the rosa create cluster command above and add a warning?
The classless inter-domain routing (CIDR) scheme of your machine must match the CIDR of your subnet.

Reply:
Well, that's not _entirely_ true. 

- "The subnet CIDRs belong to the machine CIDR that you specified." [1]
- "The VPC’s CIDR block must contain the Networking.MachineCIDR range, which is the IP address pool for cluster machines." [2]

[1] https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-validation_installing-aws-vpc
[2] https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc

I think calling out 
1. That the `--machine-cidr` flag is _required_ (since this is BYO VPC) and
2. That `machine-cidr == VPC CIDR` (this isn't a requirement, but it's a really easy way to make sure you're providing the correct CIDR)

should be the right direction. Including it in the same CLI command makes the most sense to me, but if we have another way of showing it, then that's fine as well.

Reply:
As Will points out, `--machine-cidr` is not required, but it's good to call out that this needs to match the VPC CIDR.